functable.c

```diff
@@ -148,80 +133,62 @@ static void init_functable(void) {
 
 
     // ARM - NEON
-#if defined(ARM_NEON) && defined(HAVE_BUILTIN_CTZLL)
-    if (arm_cpu_has_neon) {
-        ft.compare256 = &compare256_neon;
-        ft.longest_match = &longest_match_neon;
-        ft.longest_match_slow = &longest_match_slow_neon;
-    }
-#endif
-#ifdef ARM_NEON_ADLER32
+#ifdef ARM_NEON
 #  ifndef ARM_NOCHECK_NEON
     if (arm_cpu_has_neon)
 #  endif
+    {
         ft.adler32 = &adler32_neon;
-#endif
-#ifdef ARM_NEON_SLIDEHASH
-#  ifndef ARM_NOCHECK_NEON
-    if (arm_cpu_has_neon)
-#  endif
-        ft.slide_hash = &slide_hash_neon;
-#endif
-#ifdef ARM_NEON_CHUNKSET
-    if (arm_cpu_has_neon) {
-        ft.chunksize = &chunksize_neon;
         ft.chunkmemset_safe = &chunkmemset_safe_neon;
+        ft.chunksize = &chunksize_neon;
         ft.inflate_fast = &inflate_fast_neon;
-    }
+        ft.slide_hash = &slide_hash_neon;
+#  ifdef HAVE_BUILTIN_CTZLL
+        ft.compare256 = &compare256_neon;
+        ft.longest_match = &longest_match_neon;
+        ft.longest_match_slow = &longest_match_slow_neon;
+#  endif
+}
```

Comment:
Missing indent.

Reply:
Yup... Should be at same indentation level as line 140.

Reply:
Fixed